services/elasticloadbalancing/src/main/resources/codegen-resources/customization.config

```diff
@@ -138,7 +137,7 @@
         "TooManyAccessPointsException" : "TooManyLoadBalancersException",
         "InvalidEndPointException" : "InvalidInstanceException"
     },
-    "sendEmptyAutoConstructedListAsEmptyList" : {
+    "sendNullAsEmptyListMembers" : {
```

Comment:
We ported it over during the c2j migration for backwards compat, not sure the original motivation. To me it seems dangerous since if you forget to set the load balancers we wipe out everything.